extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js

```diff
@@ -72,7 +72,16 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',[
 					],
 					id: '9',
 					siteID: 185049
-				}
+				},
+				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)